README.md

```diff
@@ -65,13 +95,13 @@ Example API client call:
 
 ```typescript
 try {
-    // Get the current user
-    const { data } = await client.users.me()
+	// Get the current user
+	const { data } = await client.users.me()
 
-    console.log(`Hello, ${data.id}`)
+	console.log(`Hello, ${data.id}`)
 } catch ({ code, message }) {
-    // Handle error if API call failed
-    console.error(`Error fetching user: ${code} - ${message}`)
+	// Handle error if API call failed
+	console.error(`Error fetching user: ${code} - ${message}`)
```

Comment:
Could we move these back to spaces from tabs? The rendered page looks a little inconsistent with this change.

Reply:
Oops; I think this was a consequence of prettier formatting things automatically. Should I create a .prettierignore and add the documentation files to it?